src/main/webapp/pages/common/error/defaultNotFoundPage.jsp

```diff
@@ -61,7 +61,7 @@
 <body id="defaultErrorPage">
 
 <div id="header">
-    <div id="oe-logo"><img id="oe-logo-img" src="images/openelis_logo.png" title="OpenELIS" alt="OpenELIS"></div>
+    <div id="oe-logo"><img id="oe-logo-img" src="images/kapsiki-lab-logo.png" title="OpenELIS" alt="OpenELIS"></div>
```

Comment:
could you change the header to kapsiki not openelis

Reply:
Also the logOut button. can we rename it to just logOut instead of OpenElis LogOut.